### PR TITLE
repo: update the repository's issue templates

### PR DESCRIPTION
**Pull request by vince-fugnitto**
#### What it does

Fixes: #7638

The following commit updates the GitHub issue templates for the project:
- adds a bug report template.
- adds a feature request template.
- adds a `config.yml` pointing questions to 'GitHub Discussions'.

#### How to test

Review the content of the templates, suggest any modifications.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question
+    url:  https://github.com/eclipse-theia/theia/discussions
+    about: Please ask and answer questions here.
```
Comment (marcdumais-work) [inline .github/ISSUE_TEMPLATE/config.yml:5]: is the "answer" part necessary?

**Comment by marcdumais-work** [comment]
@vince-fugnitto Is there a way to try the issue templates as if the PR was merged?

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto Is there a way to try the issue templates as if the PR was merged?

Unfortunately I don't think so :(

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (vince-fugnitto) [reply .github/ISSUE_TEMPLATE/config.yml:5]: I wasn't sure what to write so I followed the `about` vscode provided.
We can however update it to anything we'd like.

Would `Please ask questions here` be more suitable?

Comment (marcdumais-work) [reply .github/ISSUE_TEMPLATE/config.yml:5]: > Would `Please ask questions here` be more suitable?

That's what I was thinking

**Comment by vince-fugnitto** [comment]
> let's try it?

Sure! We can try it out and see how it goes.

**Pull request merged**